src/pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -57,27 +57,20 @@ def __init__(
         self.device = device
         self.scaler = scaler
 
-    def pre_backward(self, model: "pl.LightningModule", closure_loss: Tensor) -> Tensor:
-        if self.scaler is not None:
-            closure_loss = self.scaler.scale(closure_loss)
-        return super().pre_backward(model, closure_loss)
-
-    def _run_backward(self, tensor: Tensor, model: Optional[Module], *args: Any, **kwargs: Any) -> None:
+    def pre_backward(self, tensor: Tensor, module: "pl.LightningModule") -> Tensor:  # type: ignore[override]
```

Comment:
There are some inconsistency with module and model associated with the `LightningModule`. Let's use model everywhere no ?

Reply:
LightningModule is a module, and in Lite we have nn.Module. This is the name that is consistent.